addon/chart-data-updater.js

```diff
@@ -15,12 +15,39 @@ export default Ember.Object.extend({
     var chart = this.get('chart');
     var self = this;
 
+	
+	//If new data has less columns
+    if (chart.datasets.length > datasets.length) {
+      var temp = Array();
+      chart.datasets.forEach(function(dataset, i) {
+		  //copy over only that amount
+        if (datasets[i]) {
+          temp.push(chart.datasets[i]);
+        }
+      });
+	  //and place into the chart data
+      chart.datasets = temp;
+    }
+
+	//If new data has more colmns
```

Comment:
Do you mind adding a test for this? I'm not sure if this will work. I think that `chart.datasets` and `datasets` have different structures.